roles/reproducer/tasks/devscripts_post.yml

```diff
@@ -35,6 +35,11 @@
             (_auth_path, 'kubeadmin-password') | ansible.builtin.path_join
           }}
 
+    - name: Ensure we expose openshift_login related facts
```

Comment:
So it may not be needed - but to be fair, I didn't have time to test without it... It doesn't really hurt at that point imho, so we should be OK, unless anyone has time to make a quick test before?

@psathyan any thoughts?

Reply:
This shouldn't block us.